tests/components/pooldose/test_sensor.py

```diff
@@ -105,16 +93,16 @@ async def test_ph_sensor_dynamic_unit(
     await hass.async_block_till_done()
 
     # Mock pH data with custom unit (should be ignored for pH sensor)
-    instant_values_raw = await async_load_fixture(hass, "instantvalues.json", DOMAIN)
-    updated_data = json.loads(instant_values_raw)
+    coordinator = mock_config_entry.runtime_data
+    updated_data = coordinator.data.copy()
```

Comment:
we shouldn't touch runtime data, that is an internal

Reply:
fixed in https://github.com/home-assistant/core/pull/156002/commits/c56811cd172f101ae7fe686414aeef9dd698befb